test/functional/test_cli_commands.py

```diff
@@ -265,6 +265,22 @@ def test_start_info_destroy_info(self):
 
         assert "Node {} runtime workspace doesn't exist".format(self.node_name) in output_info['destroy'][1]
 
+    def test_init(self):
+        """
+        CLI test: test init "-f" which will remove existing workspace
+        """
+        output_info = {}
+        output_start = run_command("infrasim node start")
+        self.assertEqual(output_start[0], 0)
+        self.assertTrue(Workspace.check_workspace_exists(self.node_name))
+        output_info['start'] = run_command("infrasim node info")
+        self.assertEqual(output_info['start'][0], 0)
+
+        output_init = run_command("infrasim init -s")
+        self.assertEqual(output_init[0], 0)
+
```

Comment:
The work space will be created after "infrasim node start", I will add a infrasim_home check.